friends/src/Features/Friends/FriendForm.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import Button from '../../App/styles/Button';
+import Card from '../../App/styles/Card';
+
+const Input = styled.input`
+  border: 1px solid #fbf7f5;
+  height: 30px;
+  border-radius: 10px;
+  padding: 10px 0;
+`;
+
+class FriendForm extends Component {
+  state = {
+    name : '',
+    age  : 0,
+    email: ''
+  };
+
+  componentDidMount() {
+    const {match, friends} = this.props;
```

Comment:
Good job on destructuring here, makes the code look much cleaner and resolves future headaches for other developers.